### ISSUE TITLE
PieChart badges are not aligned properly when pie chart wrapped in stack

### ISSUE THREAD

**Issue by Dentro2000** [imaNNeoFighT/fl_chart#538]
Hi,

I followed your suggestion to wrap pie chart in stack if I need to display text in empty centre of pie chart. Unfortunately, as I think, it causes bug with section badges alignment. 
Badges goes to proper position after tap on pie chart if you implement some touch behaviour to pie chart(for example hiding/showing text in centre after tap). 

here is sample code to reproduce:

```
import 'package:flutter/material.dart';
import 'package:fl_chart/fl_chart.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: Scaffold(body: MyHomePage(), appBar: AppBar(title: Text("BUG?"))),
    );
  }
}

class MyHomePage extends StatefulWidget {
  MyHomePage({Key key}) : super(key: key);

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return Stack(alignment: Alignment.center, children: [
      PieChart(
        PieChartData(
          sections: [
            PieChartSectionData(
              value: 12,
              color: Colors.red,
              showTitle: false,
              radius: 60,
              badgeWidget: Icon(Icons.email),
              badgePositionPercentageOffset: 0.5,
            ),
            PieChartSectionData(
              value: 12,
              color: Colors.yellow,
              showTitle: false,
              radius: 60,
              badgeWidget: Icon(Icons.error),
              badgePositionPercentageOffset: 0.5,
            )
          ],
        ),
      ),
      Text("Some Text"),
    ]);
  }
}
```
and screenshot:

![Screenshot 2021-01-08 at 16 56 00](https://user-images.githubusercontent.com/22295493/104035737-c67d1400-51d2-11eb-949a-4a441e267f78.png)

fl_chart version: 0.12.2
flutter version: Flutter 1.24.0-10.2.pre • channel beta

**Comment by Dentro2000** [comment]
anybody?

**Comment by imaNNeoFighT** [comment]
We're on it!

**Comment by imaNNeoFighT** [comment]
Fixed in [0.12.3](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.12.3).
Thanks for contributing!

**Comment by Dentro2000** [comment]
Great! Thank you!
